definition/data/sheets/CaseEventToComplexTypes.json

```diff
@@ -14,22 +14,22 @@
+    {"ID": "OtherDirection","CaseEventID": "ActionReviewComments","ListElementCode": "forParty","CaseFieldID": "otherDirections","EventElementLabel": "Who is the direction for?","EventHintText": "","FieldDisplayOrder": 3,"DisplayContext": "MANDATORY"},
+    {"ID": "OtherDirection","CaseEventID": "ActionReviewComments","ListElementCode": "directionComment","CaseFieldID": "otherDirections","EventElementLabel": "Enter the direction","EventHintText": "State what must be done, for example sending documents, confirming actions or other tasks. The directed party will see the instructions you write here.","FieldDisplayOrder": 4,"DisplayContext": "MANDATORY"},
+    {"ID": "OtherDirection","CaseEventID": "ActionReviewComments","ListElementCode": "extraDocUploadList","CaseFieldID": "otherDirections","EventElementLabel": "Add an instruction for sending documents","EventHintText": "This instruction will appear on the order as a bullet point in section 2 of 'Steps to be taken'.","FieldDisplayOrder": 5,"DisplayContext": "OPTIONAL"},
+    {"ID": "OtherDirection","CaseEventID": "ActionReviewComments","ListElementCode": "sendBy","CaseFieldID": "otherDirections","EventElementLabel": "What is the deadline for response?","EventHintText": "","FieldDisplayOrder": 6,"DisplayContext": "OPTIONAL"}, 
+    {"ID": "OtherDirection","CaseEventID": "DrawJudgesOrder","ListElementCode": "extraOrderDirection","CaseFieldID": "otherDirections","EventElementLabel": "Select direction category","EventHintText": "","FieldDisplayOrder": 1,"DisplayContext": "MANDATORY"},
+    {"ID": "OtherDirection","CaseEventID": "DrawJudgesOrder","ListElementCode": "otherDirectionHeaders","CaseFieldID": "otherDirections","EventElementLabel": "What are you ordering?","EventHintText": "","FieldDisplayOrder": 2,"DisplayContext": "MANDATORY"},
+    {"ID": "OtherDirection","CaseEventID": "DrawJudgesOrder","ListElementCode": "forParty","CaseFieldID": "otherDirections","EventElementLabel": "Who is the direction for?","EventHintText": "","FieldDisplayOrder": 3,"DisplayContext": "MANDATORY"},
+    {"ID": "OtherDirection","CaseEventID": "DrawJudgesOrder","ListElementCode": "directionComment","CaseFieldID": "otherDirections","EventElementLabel": "Enter the direction","EventHintText": "State what must be done, for example sending documents, confirming actions or other tasks. The directed party will see the instructions you write here.","FieldDisplayOrder": 4,"DisplayContext": "MANDATORY"},
+    {"ID": "OtherDirection","CaseEventID": "DrawJudgesOrder","ListElementCode": "extraDocUploadList","CaseFieldID": "otherDirections","EventElementLabel": "Add an instruction for sending documents","EventHintText":"This instruction will appear on the order as a bullet point in section 2 of 'Steps to be taken'.","FieldDisplayOrder": 5,"DisplayContext": "OPTIONAL"},
```

Comment:
This instruction will apear on the order under section 2 of 'steps to be take' as a bullet point.